include/dump_offload.hpp

```diff
@@ -180,6 +212,7 @@ class Handler : public std::enable_shared_from_this<Handler>
             unixSocket.close();
             std::remove(unixSocketPath.c_str());
         }
+        resetOffloadURI();
```

Comment:
Don't need to do this more often? What if you hit line 202?

        if (ec)
        {
            BMCWEB_LOG_ERROR << "Unix socket file clean up failed "
                             << unixSocketPath;
            this->connection->sendStreamErrorStatus(
                boost::beast::http::status::internal_server_error);
            return;
        }
        
        
        
I guess shouldn't happen